appengine-standard-java8/helloworld-gae-launch-dataflow/README.md

```diff
@@ -0,0 +1,93 @@
+HelloWorld for App Engine Standard (Java 8)
+============================
+
+This sample demonstrates how to launch a dataflow job from Google App Engine
+Standard.
+
+See the [Google App Engine standard environment documentation][ae-docs] for more
+detailed instructions.
+
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Maven](https://maven.apache.org/download.cgi) (at least 3.5)
+* [Gradle](https://gradle.org/gradle-download/) (optional)
+* [Google Cloud SDK](https://cloud.google.com/sdk/) (aka gcloud)
+* [Google Cloud Dataflow](https://cloud.google.com/dataflow/docs/)
+* [Google Cloud Dataflow Templates](https://cloud.google.com/dataflow/docs/templates/overview)
+
+## Setup
+
+1. Download and initialize the [Cloud SDK](https://cloud.google.com/sdk/)
+
+    ```
+    gcloud init
+    ```
+
+1. Create an App Engine app within the current Google Cloud Project
+
+    ```
+    gcloud app create
+    ```
+
+1. Enable the dataflow API
+
+    ```
+    gcloud services enable dataflow.googleapis.com
+    ```
+
+1. Set the project and bucket names (required for the following steps). These
+   commands add generic names; feel free to adjust appropriately.
+
+    ```
+    export PROJECT=<your-project-name-here>
+    export BUCKET=$PROJECT-template
+    ```
+
+1. Create the GCS bucket that dataflow will use
+
+    ```
+    gsutil mb gs://$BUCKET
+    ```
+
+1. Update the source to use your project and bucket names in the dataflow job.
+
+    ```
+    sed -i "s/YOUR_PROJECT_NAME/$PROJECT/" src/main/java/com/example/appengine/java8/LaunchDataflowTemplate.java
+    sed -i "s/YOUR_BUCKET_NAME/$BUCKET/" src/main/java/com/example/appengine/java8/LaunchDataflowTemplate.java
+    ```
+
+## Maven
+### Running locally
```

Comment:
Does it actually run locally?

Reply:
Yes. Looks like something with gcloud 187 is breaking the deployment though. Working on the issue now.

Reply:
Looks like downgrading to 186 fixed the issue.. going to try to file a bug
```
sudo gcloud components update --version 186.0.0
```

Reply:
here's the issue: https://stackoverflow.com/questions/48565534/failed-to-generate-quickstart-xml-while-deploying-on-app-engine-standard

Reply:
Argh - ok, mention it in the README.